backend/app/models/enrollment.rb

```diff
@@ -163,6 +163,11 @@ def subscribers
     User.where("roles && ARRAY[?]::varchar[]", roles)
   end
 
+  def already_been_copied?
+    copied_enrollment = Enrollment.where(copied_from_enrollment_id: id)
+    copied_enrollment.present?
```

Comment:
```suggestion
    Enrollment.where(copied_from_enrollment_id: id).any?
```

ça speed-up le sql ici (`any?` est traduit en SQL)